### PR TITLE
Header, Head Icon is visible and redirects

### PR DESCRIPTION
**Pull request by DaryaAndrosova**
https://trello.com/c/6DNfNp7M

### CODE REVIEW

**Review by matvlesia — COMMENTED** [review]
Hi, text 'Welcome to Jenkins!' should be moved to fixtures

File: cypress/e2e/headerHeadIcon.cy.js [hunk 1]
```diff
@@ -112,5 +112,11 @@ describe('Header Head Icon', () => {
         cy.get('div h1').should('include.text', 'Welcome to Jenkins!');
     });
 
+    it('AT_01.01.047 | Header, Head Icon is visible and redirects to home page', () => {
+        cy.get('[href="/asynchPeople/"]').click();
+        cy.get('#jenkins-head-icon').click();
+        cy.get('div h1').should('include.text', 'Welcome to Jenkins!');
```
Comment (Rom-T) [inline cypress/e2e/headerHeadIcon.cy.js:118]: Please move `Welcome to Jenkins!` in the fixtures

**Comment by NatalyaVasylyeva** [comment]
Hi! The name of your PullRequest should be the same as your AT card

File: cypress/e2e/headerHeadIcon.cy.js [hunk 2]
```diff
@@ -112,5 +112,11 @@ describe('Header Head Icon', () => {
         cy.get('div h1').should('include.text', 'Welcome to Jenkins!');
     });
 
+    it('AT_01.01.047 | Header, Head Icon is visible and redirects to home page', () => {
+        cy.get('[href="/asynchPeople/"]').click();
```
Comment (Forostinko) [inline cypress/e2e/headerHeadIcon.cy.js:116]: should be added to the fixtures

File: cypress/e2e/headerHeadIcon.cy.js [hunk 3]
```diff
@@ -112,5 +112,11 @@ describe('Header Head Icon', () => {
...
+        cy.get('#jenkins-head-icon').click();
```
Comment (Forostinko) [inline cypress/e2e/headerHeadIcon.cy.js:117]: same

**Review by SvetlanaVet — APPROVED** [review]

File: cypress/e2e/headerHeadIcon.cy.js (hunk 3 above)
Comment (Forostinko) [reply cypress/e2e/headerHeadIcon.cy.js:117]: Here same

**Review by SawkaQA — APPROVED** [review]

**Review by Rom-T — CHANGES_REQUESTED** [review]
Close please PR

**Pull request closed without merging**